components/mercury/mercury.app.js

```diff
@@ -0,0 +1,40 @@
+const axios = require("axios");
+
+module.exports = {
+  type: "app",
+  app: "mercury",
+  methods: {
+    _getBaseURL() {
+      return "https://backend.mercury.com/api/v1";
+    },
+    _getHeaders() {
+      return {
+        Authorization: `Bearer ${this.$auth.api_key}`,
+        "Content-Type": "application/json",
+      };
+    },
+    async _makeGetRequest(endpoint, params = null) {
+      const config = {
+        method: "GET",
+        url: `${this._getBaseURL()}${endpoint}`,
+        headers: this._getHeaders(),
+        params,
+      };
+      return (await axios(config)).data;
+    },
+    dayAgo() {
```

Comment:
Similar feedback: maybe rename to `daysAgo` and pass the the number of days as an arg? Or rename to `oneDayAgo()` to make the return value explicit?